### ISSUE TITLE
Result unusable with Ghost 2.0

### ISSUE THREAD

**Issue by kachurovskiy** [jessehouwing/Blogger2Ghost#13]
Jesse, huge thanks for taking time to write this tool. Sadly, looks like resulting ZIP can't be used in Ghost 2.0 (which is >1 year old):

Import failed
Please install Ghost 1.0, import the file and then update your blog to Ghost 2.0. Visit https://ghost.org/faq/upgrade-to-ghost-1-0 or ask for help in our https://forum.ghost.org.: Detected unsupported file structure.

It's understandable if you don't have time to fix this but it might be worth to put a notice somewhere. Thanks!

**Comment by kachurovskiy** [comment]
I was able to get this working by removing the ghost instance, removing the ghost_prod mysql database manually, installing ghost instance with `--v1` flag, importing the ZIP, upgrading the ghost instance then manually fixing each article.

Problems I had per-post:

- Ghost doesn't support slashes in URLs so I had to change every URL and add a `redirects.json` record for each to keep the old URLs working
- Posts weren't published so I went through every one and published it. Ghost fails with 500 on each such publish but actually does the job
- Some images weren't imported properly and I had to go back to Blogspot download them manually and insert
- For every post, manually define the featured image - usually I just removed the first image from the post and used it as featured

Overall, Blogger2Ghost saved me quite some work - many thanks!
